scopes/component/component-sizer/component-sizer.ui.runtime.ts

```diff
@@ -0,0 +1,34 @@
+// import React from 'react';
+import { DocumentNode } from 'graphql';
+import { gql } from '@apollo/client';
+import { ComponentAspect, ComponentUI } from '@teambit/component';
+import { UIRuntime } from '@teambit/ui';
+import { DocsAspect, DocsUI } from '@teambit/docs';
+import { ComponentSizerAspect } from './component-sizer.aspect';
+
+/**
+ * Component code tab aspect. Presents the code tab page and allows to control the code tab and register specific icons for each file type.
```

Comment:
This refers to the wrong aspect

Reply:
currently doesnt really do much. this will be used to register the size to the docs aspect